tests/common/dualtor/dual_tor_mock.py

```diff
@@ -92,18 +92,18 @@ def _apply_dual_tor_state_to_orchagent(dut, state):
 
 
 @pytest.fixture(scope='module')
-def apply_active_state_to_orchagent(duthosts, rand_one_dut_hostname):
-    dut = duthosts[rand_one_dut_hostname] 
+def apply_active_state_to_orchagent(duthosts, rand_one_dut_hostname, tor_mux_intfs):
+    dut = duthosts[rand_one_dut_hostname]
```

Comment:
Is this func going to have the same behavior as before? Here we are going to get any dut based on rand_one_dut_hostname but tor_mux_intfs fixture makes use of the first index in duthosts.

Reply:
The `tor_mux_intfs` fixture just return a list of interface names that are connected to mux cable. The list is the same for different duthost in dual tor topology.